src/components/Navbar.js

```diff
@@ -0,0 +1,16 @@
+import { NavLink, Outlet } from 'react-router-dom';
+
+const Navbar = () => (
+  <>
+    <nav>
+      <h1>Math Magicians</h1>
+      <div>
+        <NavLink to="/">Home</NavLink>
+        <NavLink to="/quote">Quote</NavLink>
+        <NavLink to="/calculator">Calculator</NavLink>
+      </div>
+    </nav>
+    <Outlet />
```

Comment:
Hi @kazim110

Great job creating multiple pages with dedicated routes made possible by [React Router V6](https://reactrouter.com/). Remember that the general layout of the page should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

In your case, the order of the nav items does not match the wireframe. On your page, the order is ` Home | Quote | Calculator` when it should be `Home | Calculator | Quote`.

![image](https://github.com/kazim110/math-magicians2/assets/38088818/c5e00eae-e7db-428c-8e10-6ae021b208c6)

![image](https://github.com/kazim110/math-magicians2/assets/38088818/42f46749-65b0-4f41-9b14-05ec206bb2c4)